_includes/default-header.html

```diff
@@ -19,40 +19,33 @@
       <div id="top-menu" class="navbar-menu">
         <div class="navbar-end">
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link" href="{% link openseeds/index.md %}">Open Seeds program</a>
+            <a class="navbar-link" href="{% link index.md %}">OLS Pillars</a>
             <div class="navbar-dropdown">
-              <a class="navbar-item" href="{% link openseeds/index.md %}#applications"> Applications </a>
-              <a class="navbar-item" href="{% link about.md %}"> The program itself </a>
-              <a class="navbar-item" href="{% link about.md %}#projects"> Projects </a>
-              <a class="navbar-item navbar-previous-title" href="{% link about.md %}#the-roles"> Role description </a>
-              <a class="navbar-item" href="{% link about.md %}#mentees"> Mentees </a>
-              <a class="navbar-item" href="{% link about.md %}#mentors"> Mentors </a>
-              <a class="navbar-item" href="{% link about.md %}#experts"> Experts </a>
-              <a class="navbar-item" href="{% link about.md %}#facilitators"> Facilitators </a>
-              <a class="navbar-item navbar-previous-title" href="{% link openseeds/index.md %}#cohorts"> Cohorts </a>
-              <a class="navbar-item navbar-previous-title" href="{% link library.md %}"> Video Library </a>
+              <a class="navbar-item navbar-previous-title" href="#">Open Science Training</a>
+              <a class="navbar-item" href="{% link openseeds/index.md %}">Open Seeds Program</a>
+              <a class="navbar-item" href="{% link _posts/2023-05-05-NASA-grant-announcement.md %}">NASA TOPS</a>
+              <a class="navbar-item navbar-previous-title" href="#">Research on Open</a>
+              <a class="navbar-item" href="#">WT impact research</a>
+              <a class="navbar-item" href="#">Turing Skills Policy Award</a>
+              <a class="navbar-item navbar-previous-title" href="#">Open Incubator</a>
+              <a class="navbar-item" href="#">Resident fellows program</a>
```

Comment:
There is no section in the `open-incubator.md` file to link to that

Reply:
About that...

You're right. The individual pages for the 3 Pillars still need more content. You may have noticed in PR #688 that only `open-research.md` seems fully developed. The other two? Not so much.

I think Yo is in the process of writing the content to be added.

Until we can get more information added to `open-incubator.md`, we probably have two options:

1. Merge this PR as is (considering all that is left is what Yo will give us).
2. Wait for PR 688 to be complete, and link appropriately before merging this.

What do you think?

Reply:
P.S. The tests are failing because we don't have the `open.research.md`, etc mentioned in the navbar. It should be fine once the other PR is ready (or we can remove the links and use # for now).

Basically, no cause for alarm. 😄

Reply:
I would wait to merge this PR than the other one is merged so everything could be fixed and in place